omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -125,6 +127,13 @@ public String showForm(@RequestParam(required = false, value = "userId") Integer
 			model.addAttribute("secretQuestion", userService.getSecretQuestion(user));
 		}
 		
+		if (user.getPerson().getId() != null && !Context.getProviderService().getProvidersByPerson(user.getPerson()).isEmpty()) {
+			model.addAttribute("isProvider", true);
+			model.addAttribute("providerList", new ArrayList<Provider>(Context.getProviderService().getProvidersByPerson(user.getPerson())));
```

Comment:
You are right. It is not needed. I have removed it. I will commit the latest code